{{cookiecutter.project_shortname}}/Dockerfile

```diff
@@ -1,13 +1,17 @@
+# This is the Dockerfile that builds a fully functional image of your app
 {% include 'misc/header.py' %}
 ARG DEPENDENCIES_VERSION=latest
+# Get the previously built image(Docker.base) with pip dependencies and extend it
 FROM {{cookiecutter.project_shortname}}-base:${DEPENDENCIES_VERSION}
 
 COPY ./ .
 COPY ./docker/uwsgi/ ${INVENIO_INSTANCE_PATH}
+# install the app
+RUN pip install .
```

Comment:
why are we using pip here, and in Dockerfile.base we use ``pipenv``?

Reply:
Here we are installing the app while in Dockerfile.base we are installing the dependencies that are stored in the Pipfile so we need to use pipenv to install them.